package.json

```diff
@@ -34,10 +34,12 @@
     "classnames": "^2.2.5",
     "core-js": "^2.4.1",
     "cron": "^1.3.0",
+    "dexie": "^2.0.1",
     "diff-match-patch": "^1.0.0",
     "docuri": "^4.2.2",
     "fathom-web": "^2.1.0",
     "history": "^4.6.3",
+    "indexeddb-export-import": "^1.0.3",
```

Comment:
make sure to `yarn remove` these packages if not being used anymore. Also noticed there is a `package-lock.json` committed. We're already using yarn to manage sub-level dep versions, this is just npm 5's version of this, so no need to commit